app/src/main/java/com/sun_asterisk/moviedb_50/screen/details/MovieDetailsFragment.kt

```diff
@@ -112,6 +119,27 @@ class MovieDetailsFragment : Fragment(), MovieDetailsContract.View {
         trailerAdapter.updateData(movieTrailers)
     }
 
+    override fun showFavoriteImage(type: String) {
+        if (type == Constant.BASE_NOTIFY_ADD_FAVORITE_SUCCESS) {
+            view?.favoriteImageView?.setImageResource(R.drawable.ic_favorite)
+        } else {
+            view?.favoriteImageView?.setImageResource(R.drawable.ic_favorite_border)
+        }
+    }
+
+    override fun notifyFavorite(type: String) {
+        when (type) {
+            Constant.BASE_NOTIFY_ADD_FAVORITE_SUCCESS ->
+                activity?.showSnackBar(R.string.notification_add_favorite_success)
+            Constant.BASE_NOTIFY_ADD_FAVORITE_ERROR ->
+                activity?.showSnackBar(R.string.notification_add_favorite_failed)
+            Constant.BASE_NOTIFY_DELETE_FAVORITE_SUCCESS ->
+                activity?.showSnackBar(R.string.notification_delete_success)
+            Constant.BASE_NOTIFY_DELETE_FAVORITE_ERROR ->
+                activity?.showSnackBar(R.string.notification_delete_failed)
+        }
+    }
```

Comment:
```suggestion
        when (type) {
            FavoriteResultType.addSucces -> activity?.showSnackBar(R.string.notification_add_favorite_success)
        }
    }
```

tương tự các case tiếp theo